Doc/Manual/Go.html

```diff
@@ -60,38 +60,44 @@
 
 
 <p>
-Go is a compiled language, not a scripting language.  However, it does
-not support direct calling of functions written in C/C++.  The cgo
-program may be used to generate wrappers to call C code from Go, but
-there is no convenient way to call C++ code.  SWIG fills this gap.
+Go does not support direct calling of functions written in C/C++.  The
+<a href="https://golang.org/cmd/cgo/">cgo program</a> may be used to generate
+wrappers to call C code from Go, but there is no convenient way to call C++
+code.  SWIG fills this gap.
 </p>
 
 <p>
-There are (at least) two different Go compilers.  One is the gc
-compiler, normally invoked via the go tool.  The other
-is the gccgo compiler, which is a frontend to the gcc compiler suite.
-The interface to C/C++ code is completely different for the two Go
-compilers.  SWIG supports both, selected by a command line option.
+There are (at least) two different Go compilers.  The first is the Go compiler
+of the <a href="https://golang.org/doc/install">Go distribution</a>.
+Since Go 1.5 the Go compiler is part of the <a href="https://golang.org/cmd/go/">
```

Comment:
Well, there was never a gc binary.  There was a 6g binary, an 8g binary, etc.  In 1.5 those have all been combined into a single compile binary.  So where you used to run "go tool 6g" you can now run "go tool compile".

But when discussing Go compilers, the overall name of the compiler that is part of the standard Go distribution is "gc", whereas the name of the compiler that is part of the GCC distribution is "gccgo".

I feel like the original text of this paragraph is accurate.  It's fine to add some links but I don't see a great need to change the text.  In particular nothing in the original paragraph is different in the 1.5 release.